src/components/body/BodyMainPanel.tsx

```diff
@@ -45,20 +57,54 @@ const BodyMainPanel: React.FC<IProps> = ({
 
   const contentContainerStyle = React.useMemo(
     () => ({
-      paddingLeft: styleLeft,
-      paddingTop: styleTop,
+      top: styleTop,
+      left: styleLeft,
+      paddingLeft: _scrollLeft,
+      paddingTop: _scrollTop,
       height: dataLength * bodyRowHeight,
       width: bodyContentWidth,
     }),
-    [styleTop, styleLeft, dataLength, bodyRowHeight, bodyContentWidth]
+    [
+      styleTop,
+      styleLeft,
+      _scrollTop,
+      _scrollLeft,
+      dataLength,
+      bodyRowHeight,
+      bodyContentWidth,
+    ]
   );
 
+  const onScroll: React.UIEventHandler<HTMLDivElement> = throttle(
+    React.useCallback(() => {
```

Comment:
```suggestion
    React.useCallback( throttle(() => {
```

Reply:
Please move throttle inside useCallback.
After then, check deps for useCallback.